### ISSUE TITLE
Integrate with SendGrid

### ISSUE THREAD

**Issue by sunezapa** [sendgrid/docs#1793]
https://sendgrid.com/docs/Integrate/index.html

Regard "The IP addresses at smtp.sendgrid.net are changed often and without notice. If you point your traffic to one specific IP, you will experience interruptions in your service when these IPs are changed".

There is a problem by using "smtp.sendgrid.net" as only information for SMTP-connection:
Many providers has blocked access to extermal SMTP-servers, for security.
SiteGround for example do block - but it is easy to ask the support to whitelist IP's for external SMTP, but it can only be done for IP's, not for URL. It is possible to whitelist RANGES, as for Googles SMTP, but it is needed to what range(s)!
I have looked for IP-range information with no success.
Without this info SendGrid will not work by those providers, though making the recommended solution unuseable.
The platform is Magento, using  AScroeder's SMTP Pro Email Extension

**Comment by mbernier** [comment]
Are you able to use an API to send email? If so, then you could use our mail API instead of SMTP and the traffic would go over port 443 instead of port 25, meaning that the server won't block that traffic.

**Comment by sunezapa** [comment]
I noticed the API option,
but I am not aware of any way to use API with Magento for transactional emails from the shop?

 

That's why I use AScroeder's SMTP PRO plugin.
It works with gmail, whitelisting (several) IP-ranges, that I got info about from Google.
But now we meet the max. 500 emails/day limit in Google,

and test SendGrid as a solution.

I am aware that we need to pay for SendGrid, but await confirmation that it will work ;-)

 

…sune…
